playground/backend/internal/errors/grpc_errors_test.go

```diff
@@ -22,21 +22,28 @@ import (
 
 func TestInternalError(t *testing.T) {
 	type args struct {
-		title   string
-		message string
+		title         string
+		formatMessage string
+		arg           interface{}
 	}
 	tests := []struct {
 		name     string
 		args     args
 		expected string
 		wantErr  bool
 	}{
-		{name: "TestInternalError", args: args{title: "TEST_TITLE", message: "TEST_MESSAGE"},
-			expected: "rpc error: code = Internal desc = TEST_TITLE: TEST_MESSAGE", wantErr: true},
+		{
+			// Test case with calling InternalError method.
+			// As a result, want to receive error with correct error message.
+			name:     "TestInternalError",
```

Comment:
Do we have some negative test cases here? Like, for example, the case when the `formatMessage` doesn't match the number of `args`?

Reply:
Added tests.